akka-projection-slick/src/main/scala/akka/projection/slick/internal/SlickOffsetStore.scala

```diff
@@ -113,7 +113,7 @@ import slick.jdbc.JdbcProfile
 
   val offsetTable = TableQuery[OffsetStoreTable]
 
-  def createIfNotExists: Future[Done] = {
+  def createIfNotExists(): Future[Done] = {
```

Comment:
side-effect method, using `()`. 

This is internal API, so no big deal to change it.